src/main/java/cubicchunks/CubicChunks.java

```diff
@@ -330,7 +330,10 @@ public int getValue() {
             USE_VANILLA_CHUNK_WORLD_GENERATORS(false,
                     "Enabling this option will force " + CubicChunks.MODID
                             + " to use world generators designed for two dimensional chunks, which are often used for custom ore generators added by mods. To do so "
-                            + CubicChunks.MODID + " will pregenerate cubes in a range of height from 0 to 255.");
+                            + CubicChunks.MODID + " will pregenerate cubes in a range of height from 0 to 255."),
+            FORCE_CUBIC_CHUNKS(false,
+                    "Enabling this will force creating a cubic chunks world, even if it's not cubic chunks world type. This option is automatically"
+                            + " set in world creation GUI when creating cubic chunks world with non-cubicchunks world type");
```

Comment:
I don't get it.

> This option is automatically set in world creation GUI when creating cubic chunks world with non-cubicchunks world type

Do you mean: "If set to *true*, launching custom non-cubicchunks world creation settings GUI will show options available for cubic chunks worlds"?